<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Time_table bot</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>700</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Timetable</string>
     </attribute>
     <widget class="QTableWidget" name="timetable">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>461</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="lessons_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="db_update">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>31</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>|&gt;</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="add_time">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lesson_line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Урок</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_lesson">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="authorize">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>480</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Авторизация</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>VK_api</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;АВТОРИЗАЦИЯ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditToken">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>230</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>35</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Token</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditId">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>80</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>35</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Club_id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OkButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
